editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5682,7 +5670,7 @@ CanvasItemEditor::CanvasItemEditor() {
 	set_process_shortcut_input(true);
 	clear(); // Make sure values are initialized.
 
-	// Update the menus' checkboxes.
+	// Update the menu's checkboxes.
```

Comment:
Is it just one menu? Otherwise this is incorrect, `menus'` means "belonging to several menus"